### ISSUE TITLE
re-implement home page template

### ISSUE THREAD

**Issue by ghwood** [ietf-tools/wagtail_website#336]
the current www.ietf.org home page ought to be reconsidered and updated.

A full specification is yet to be developed, but this would include rethinking the content based on recent information available from analytics about how the site is used, and a refined understanding of what is most important to highlight for key audiences coming to the site.

**Comment by cdanfon** [comment]
Hey @ghwood same for this one, could you share specification requirements and a prototype that @mgax can work from? Thanks!

**Comment by ghwood** [comment]
Will do. Note that this will take some time as we are just starting on the specification.

**Comment by cdanfon** [comment]
Hi @ghwood thanks for the heads up. For planning purposes, would it be ready on time for Alex to implement - his last day supporting IETF is Tues 19th Dec. If that's not going to be the case we can always look at other tickets in the backlog.

**Comment by ghwood** [comment]
Got it. We are working on this now and have further detail by the end of the week (1 Dec) so can assess then.

**Comment by cdanfon** [comment]
Hey @ghwood any news on this? Thanks!

**Comment by ghwood** [comment]
Yes, will have more detail today (I've run into an issue with our cloud-based toolset).

**Comment by ghwood** [comment]
This is intended to be a straight-forward update to the home page to streamline and improve it homepage for the vast majority of visitors by simply removing the “below the fold” part of the page that has very, very little interaction. In particular, these re the "Understanding the Internet Engineering Task Force" and "Topics of interest" sections. Screenshots of the desired outcome are attached here and can be found on the [design board](https://miro.com/app/board/uXjVNJMns3E=/?share_link_id=921401563596). 
<img width="564" alt="image" src="https://github.com/ietf-tools/wagtail_website/assets/5347540/bfa16319-c427-4690-aecf-392b127b747a">
<img width="101" alt="image" src="https://github.com/ietf-tools/wagtail_website/assets/5347540/94026f54-287a-451d-be5e-a656e52b4bf3">

**Comment by cdanfon** [comment]
Thanks Greg! We will implement the above change as part of this ticket.

**Comment by mgax** [comment]
Hi @ghwood, just a quick check-in to see whether you had a chance to clarify what the homepage should look like.

**Comment by ghwood** [comment]
Hi @mgax We have not gone further than the interim approach already on staging and anticipated for deployment on production in a few weeks.

**Comment by cdanfon** [comment]
Hey @ghwood this ticket is marked with the label 'contractor'. Is there anything you need Alex to do, or can we consider this not something Alex needs to work on based on your comment? Thanks

**Comment by ghwood** [comment]
Correct @cdanfon this does not need action at the moment. We'll update the ticket when we have something further to consider.
